cluster-assign-roles.sh

```diff
@@ -34,7 +34,7 @@ KNIFE_ADMIN="-u admin -k /etc/chef-server/admin.pem"
 # Argument: $1 - a string of role!IP!FQDN pairs separated by white space
 # Will install the machine with role $role in the order passed (left to right)
 function install_machines {
-  passwd=`sudo knife data bag show configs $ENVIRONMENT $KNIFE_ADMIN | grep "cobbler-root-password:" | awk ' {print $2}'`
+  passwd=`sudo knife vault show os cobbler "root-password" --mode client | grep "root-password:" | awk ' {print $2}'`
```

Comment:
sudo knife vault show os cobbler "root-password" --mode client
because this will return something like:
id:            cobbler
root-password: ************

So we still need grep "root-password" to strip id item out.